journal/modernized/37-chapter-34.adoc

```diff
@@ -991,11 +994,17 @@ Daniel Wheeler
 [.signed-section-context-close]
 Henry Freelng, of Paihia, Bay of Islands, First Month 18th, 1837.
 
+--
+
 [.offset]
 A copy of a letter from the missionaries at Paihia
 
+[.embedded-content-document.letter]
+--
+
 [.signed-section-context-open]
 Church Missionary Society`'s Station, Paihia, Bay of Islands, New Zealand, 18th January, 1837.
+//Does the linter catch this after i submit the PR? or should it have caught it so I can break up line 1006 now?
```

Comment:
Yeah, this line is over 90, but under 100, so it's fine.  If you don't see the linter barking about it, it should be fine. I may have never mentioned this, but I fixed all the lines in all the books that were over 100 characters, so the lint rule is really to help not accidentally introduce _new_ long lines while editing. 👍